### PR TITLE
modify ``val_loss_mean``

### PR DESCRIPTION
**Pull request by xujiaze13**
revised by this warning
***/lib/python3.*/site-packages/pytorch_lightning/utilities/distributed.py:25: RuntimeWarning: The metric you returned 1.234 must be a `torch.Tensor` instance, checkpoint not saved HINT: what is the value of val_loss in validation_epoch_end()?

### CODE REVIEW

**Review by JetRunner — APPROVED** [review]
Good catch!
